tools/Init2.ps1

```diff
@@ -0,0 +1,32 @@
+	##copy "E:\Git\PS\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Resources\Microsoft.Azure.Commands.ResourceManager.Cmdlets.*" "C:\Program Files\WindowsPowerShell\Modules\AzureRM.Resources\0.10.0\"
```

Comment:
@rnikhilkumar If you need to store a credential securely, for demo purposes, you might consider using the Windows Data Protection API (DPAPI). /cc @markcowl 

I also strongly recommend avoiding the use of positional parameters for public-facing code. The use of C#-like syntax in conjunction with the `New-Object` cmdlet is unfortunately prevalent, but it's important to teach people the correct method of using the command.

```
### Store password using DPAPI
(Get-Credential).Password | ConvertFrom-SecureString | Out-File -FilePath c:\temp\encryptedpassword.txt;

### Retrieve password using DPAPI
$Password = Get-Content -Path c:\temp\encryptedpassword.txt | ConvertTo-SecureString;

### Construct the PSCredential instance
$Username = 'username@domain.com';
$Credential = New-Object -TypeName PSCredential -ArgumentList $Username, $Password;
```

Cheers,
Trevor Sullivan
Microsoft MVP: PowerShell;